tests/mechanic/telemetry_test.py

```diff
@@ -295,20 +295,14 @@ def test_stores_default_ccr_stats(self, metrics_store_put_count):
             "shard": '0'
         }
 
-        metrics_store_put_count.assert_has_calls([
-            mock.call(name="total_fetch_time_millis", count=total_fetch_time_millis, unit="ms", meta_data=shard_metadata),
-            mock.call(name="total_index_time_millis", count=total_index_time_millis, unit="ms", meta_data=shard_metadata),
-            mock.call(name="operations_received_field", count=operations_received_field, meta_data=shard_metadata),
-            mock.call(name="number_of_batches_field", count=number_of_batches_field, meta_data=shard_metadata),
-            mock.call(name="total_transferred_bytes", count=total_transferred_bytes, unit="byte", meta_data=shard_metadata),
-            mock.call(name="current_idle_time_millis", count=current_idle_time_millis, unit="ms", meta_data=shard_metadata),
-            mock.call(name="leader_max_seq_no", count=leader_max_seq_no, meta_data=shard_metadata),
-            mock.call(name="follower_primary_max_seq_no", count=follower_primary_max_seq_no, meta_data=shard_metadata),
-            mock.call(name="processed_global_checkpoint", count=processed_global_checkpoint, meta_data=shard_metadata),
-        ], any_order=True)
+        metrics_store_put_doc.assert_called_with(
+            ccr_stats_follower_response["follower"]["0"],
+            level=MetaInfoScope.cluster,
+            meta_data=shard_metadata
+        )
 
-    @mock.patch("esrally.metrics.EsMetricsStore.put_count_cluster_level")
-    def test_stores_default_ccr_stats_many_shards(self, metrics_store_put_count):
+    @mock.patch("esrally.metrics.EsMetricsStore.put_doc")
+    def test_stores_default_ccr_stats_many_shards(self, metrics_store_put_doc):
         shard_range = range(2)
         total_fetch_time_millis = [random.randint(0,9999999) for i in shard_range]
```

Comment:
nit: missing space after comma. Also, we could use `_` instead of a variable name here? i.e. `[random.randint(0, 9999999) for _ in shard_range]`